io.github.Strozer2002.sparrow.yaml

```diff
@@ -0,0 +1,30 @@
+app-id: io.github.Strozer2002.sparrow
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: sparrow
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+modules:
+  - name: sparrow
+    buildsystem: simple
+    build-commands:
+      - cp -R bin/ /app/sparrow
+      - mkdir -p /app/sparrow
+      - chmod +x /app/sparrow/sparrow
+      - mkdir /app/bin/
+      - ln -s /app/sparrow/sparrow /app/bin/sparrow
+      - install -Dm644 io.github.Strozer2002.sparrow.metainfo.xml /app/share/metainfo/io.github.Strozer2002.sparrow.metainfo.xml
+      - install -Dm644 icons/128x128/io.github.Strozer2002.sparrow.png /app/share/icons/hicolor/128x128/apps/io.github.Strozer2002.sparrow.png
+      - install -Dm644 icons/256x256/io.github.Strozer2002.sparrow.png /app/share/icons/hicolor/256x256/apps/io.github.Strozer2002.sparrow.png
+      - install -Dm644 io.github.Strozer2002.sparrow.desktop /app/share/applications/io.github.Strozer2002.sparrow.desktop
```

Comment:
Exec is `Exec=io.github.Strozer2002.sparrow` it should be `Exec=sparrow` in desktop file

Reply:
resolved